wiki/Article_styling_criteria/en.md

```diff
@@ -0,0 +1,23 @@
+---
+tags:
+  - wiki standards
+---
+
+# Article styling criteria
+
+*For news posts, see: [News Styling Criteria](/wiki/News_Styling_Criteria)*
+
+The article styling criteria (ASC) serves as the osu! wiki's enforced styling standards to keep consistency in clarity, formatting, and layout in all articles.
+
+All articles should aim at having proper grammar, correct spelling, and correct information.
+
+If you have any suggestions regarding the article styling criteria, [open an issue on GitHub](https://github.com/ppy/osu-wiki/issues/new).
```

Comment:
```suggestion
To suggest changes regarding the article styling criteria, [open an issue on GitHub](https://github.com/ppy/osu-wiki/issues/new "GitHub").
```
Prob don't want to break the ASC rules within the ASC. Dismiss this suggestion if this was an intentional use of informal register.